src/index.js

```diff
@@ -43,16 +47,22 @@ addListBtn.addEventListener('click', (e) => {
     inputValue.value = '';
     displayTodo();
     classObject.removeTodo();
-
     for (let i = 0; i < classObject.list.length; i += 1) {
       n = i + 1;
       classObject.list[i].index = n;
     }
-
     classObject.addToStorage(classObject.list);
   }
 });
 
+classObject.generatedList.addEventListener('click', classObject.deleteDone);
+clearAll.addEventListener('click', () => {
+  classObject.clear();
+  displayTodo();
+
+  // checkBox.parentElement.innerHTML = '';
```

Comment:
- Kindly remove all commented-out lines of code to make your code clean :recycle: